### PR TITLE
feat/softplus: Add softplus function

### PR DESCRIPTION
**Pull request by ojetokun**
Added softplus activation function
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
Added softplus activation function

Issue Number: N/A

## What is the new behavior?

`NN::softplus(@tensor)`

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by raphaelDkhn** [comment]
@ojetokun can you run please scarb run docgen to generate documentation from your docstrings ?

**Comment by franalgaba** [comment]
Hi @ojetokun ! Amazing work. Only a few things are missing:

- Docs for the new function using `scarb run docgen`.
- Update the `docs/apis/compatibility.md` file with the new addition and compatibility %.

**Comment by ojetokun** [comment]
@franalgaba @raphaelDkhn Done. 
Side note: The last line in the compatibility.md doc might need to be updated when merging this PR or #65

File: docs/apis/numbers/fixed-point/README.md [hunk 1]
```diff
@@ -35,23 +35,23 @@ use orion::numbers::fixed_point::Fixed;
...
-| [`fp.ln`](fp.ln.md)                                   | Returns the natural logarithm of the fixed point number.                                        |
-| [`fp.log2`](fp.log2.md)                               | Returns the base-2 logarithm of the fixed point number.                                         |
-| [`fp.log10`](fp.log10.md)                             | Returns the base-10 logarithm of the fixed point number.                                        |
-| [`fp.pow`](fp.pow.md)                                 | Returns the result of raising the fixed point number to the power of another fixed point number |
-| [`fp.round`](fp.round.md)                             | Rounds the fixed point number to the nearest whole number.                                      |
-| [`fp.sqrt`](fp.sqrt.md)                               | Returns the square root of the fixed point number.                                              |
+| function | description |
+| --- | --- |
+| [`fp.new`](https://orion.gizatech.xyz/apis/numbers/fixed-point/fp.new.md) | Constructs a new `FixedType` instance. |
```
Comment (franalgaba) [inline docs/apis/numbers/fixed-point/README.md:40]: Do we want to replace all references of the docs with links? @raphaelDkhn

**Review by franalgaba — COMMENTED** [review]
Everything looks good from my side. Only some clarity about link references in the docs :)

File: docs/apis/operators/tensor/tensor.new.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# tensor.new
+# TensorTrait::new
```
Comment (franalgaba) [inline docs/apis/operators/tensor/tensor.new.md:1]:
```suggestion
# tensor.new
```

**Pull request merged**